loadtest/control/simulcontroller/actions.go

```diff
@@ -1282,3 +1282,365 @@ func sendTypingEventIfEnabled(u user.User, channelId string) error {
 	}
 	return nil
 }
+
+func (c *SimulController) viewGlobalThreads(u user.User) control.UserActionResponse {
+	collapsedThreads, resp := control.CollapsedThreadsEnabled(u)
```

Comment:
I know this logic already exists, but do we have a configurable ratio for users with CRT enabled vs. disabled? It would be neat to test at 100%, 50%, etc.

Reply:
Currently we don't have a way to to configure the ratio. Every user is set to use CRT through the `default_on` value for the `ServiceSettings.CollapsedThreads` server setting.

This is definitely an improvement I'd like to make in the future. Currently, even with CRT off, the global frequency of actions takes into account the frequency of CRT actions when choosing which action to execute. This check makes the CRT action a no-op.
Ideally, if CRT is off, the action chooser should not consider CRT actions as valid actions in the list of possible actions.